src/index.js

```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```

Comment:
It's a common notation for unused variables. Eslint allow [using a pattern](http://eslint.org/docs/rules/no-unused-vars#argsignorepattern) to ignore those. When configured correctly, Eslint doesn't complain about it.